chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -64,7 +64,7 @@
 # the server.  In most cases, the default value should be fine.
 #
 # tunnel_bridge = br-tun
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") || @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
> if (@ml2_type_drivers & %w{gre vxlan}).any?

I dislike this, the current code is better readable especially as many people don't know what the ```&``` operator does in this case.